tldw_Server_API/tests/AuthNZ/integration/test_single_user_claims_permissions.py

```
@@ -0,0 +1,240 @@
+            org_ids=[],
+            team_ids=[],
+        )
+        try:
+            request.state.auth = AuthContext(
+                principal=principal,
+                ip="127.0.0.1",
+                user_agent="pytest-agent",
+                request_id=request_id,
+            )
+        except Exception:
+            pass
```

Ruff (0.14.6)

`try`-`except`-`pass` detected, consider logging the exception

(S110)